frontend/src/metabase-types/api/settings.ts

```diff
@@ -203,6 +203,7 @@ export interface Settings {
   "custom-formatting": FormattingSettings;
   "custom-homepage": boolean;
   "custom-homepage-dashboard": number | null;
+  "default-browse-tab": string | null;
```

Comment:
let's get more specific here 

```suggestion
  "default-browse-tab": "models" | "databases" | null;
```

Reply:
I thought about that. Since this constraint isn't imposed on the server, and this is where we store the server's value, it seemed safer not to assume we know anything about what the value is. I have a function that checks if the value is valid and ignores it, if it isn't.